portfolio/src/main/webapp/index.html

```diff
@@ -1,17 +1,263 @@
+              HTML, CSS, and Javascript</a
+            >
+          </li>
+          <li>
+            <a
+              href="https://medium.com/@9cv9official/what-are-get-post-put-patch-delete-a-walkthrough-with-javascripts-fetch-api-17be31755d28"
+              >What are GET, POST, PUT, PATCH, DELETE? A walkthrough with
+              JavaScript’s Fetch API.</a
+            >
+          </li>
+          <li>
+            <a
+              href="https://medium.com/@9cv9official/building-a-rest-api-with-django-rest-framework-fabb83ef35e3"
+              >Building a REST API with Django Rest Framework</a
+            >
+          </li>
+        </ul>
+      </section>
+      <hr class="solid">
+      <section id="interests">
+        <h1>My Interests & Hobbies</h1>
+        <p>
+          In my spare time I love playing tennis! I started learning when I was
+          about 12 back in my home country Cambodia. After coming to Singapore,
+          I played in my high school team and while I was at Tembusu College I
+          played with the college team too! Another thing I love is Netflix.
+          When I want to relax, Netflix and my bed are my go-to place.
+          Otherwise, I'd take out my guitar and start strumming!
+        </p>
+        <p>
+          One hobby I really wish to pick up is SCUBA diving! In the summer of
+          2019 when I was living in Indonesia, I did a fun dive in the Gili
+          islands and instantly fell in love with the underwater world. You're
+          weightless, probably much like floating in space... When travel
+          becomes somewhat normal again I plan to get my SCUBA licenses and go
+          diving anywhere I can afford.
+        </p>
+        <p>
+          Also, last summer, I also bought an old film camera and just pointed and
+          shoot stuff. Here are some pictures that developed quite nicely! There
+          was one roll where I accidentally used the same roll twice and had
+          some accidental double exposures.
+        </p>
+        <button onclick="addFilmImages()">View Image Gallery</button>
+        <div id="image-gallery-container"></div>
+      </section>
+    </main>
+
+    <footer>
+      <div class="social">
+        <div class="tooltip">
+          <a id="social-icon" href="https://github.com/dalisc/"
+            ><i class="fab fa-github icon-3x"></i
+          ></a>
+          <span class="tooltiptext">GitHub profile</span>
+        </div>
+
+        <div class="tooltip">
+          <a id="social-icon" href="https://www.linkedin.com/in/dalisc/"
+          ><i class="fab fa-linkedin"></i
+        ></a>
+          <span class="tooltiptext">LinkedIn profile</span>
+        </div>
+        <div class="tooltip">
+          <a id="social-icon" href="mailto:dalischan97@gmail.com"
+          ><i class="fas fa-envelope-open-text"></i
+        ></a>
+          <span class="tooltiptext">Email me</span>
+        </div>
+
+
+      </div>
+    </footer>
+
+    <script>
```

Comment:
Consider moving this to a separate javascript file :)

Reply:
I had a problem where when i moved it and imported it the javascript ended up not working :( i'll look into this issue and try again

Reply:
i have moved it to a new file. turns out i had to wrap in a window.onload function :D